### ISSUE TITLE
Selectors are not WCAG-compliant

### ISSUE THREAD

**Issue by alansemenov** [enonic/lib-admin-ui#3924]
Selectors should have the following attributes:
* `role="combobox"`
* `aria-haspopup="listbox"`
* `aria-expanded` (that toggles between `true` or `false` based on whether the dropdown is visible or not)

Selectors' dropdown should have:
* `role="listbox"`
* `aria-hidden` (that toggles between `true` or `false` based on whether the dropdown is visible or not)

Each item in the dropdown list should have:
* `role="option"`
* `aria-selected="true"` (if the item is selected in the dropdown)
